scout/server/blueprints/institutes/controllers.py

```diff
@@ -606,7 +606,10 @@ def update_variant_genes(store, variant_obj, genome_build):
 
     hgvs_c = []
     hgvs_p = []
-    gene_symbols = set()
+    gene_symbols = list()
+    canonical_transcripts = []
```

Comment:
Why not 
gene_symbols = []

Like those below?

Reply:
Ah, just adhering to the style, temporarily! 😊